page.codeberg.JakobDev.CCLite.yaml

```diff
@@ -0,0 +1,93 @@
+id: page.codeberg.JakobDev.CCLite
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: cclite
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --env=LUA_PATH=./?.lua;/app/share/luajit-2.1.0-beta3/?.lua;/app/share/lua/5.1/?.lua;/app/share/lua/5.1/?/init.lua
+  - --env=LUA_CPATH=./?.so;/app/lib/lua/5.1/?.so;/app/lib/lua/5.1/loadall.so
```

Comment:
these two should be set in the launcher script `cclite.sh` instead.

Reply:
I choose this Way, because I can test it in the interactive Lua prompt. But for the end user, it does not matter if the variables are set in the launch script or in the Manifest.

Reply:
I don't see why it is a bad thing, that the User can override this, but I had it moved now to the launch script